test/suites/standard/version.robot

```diff
@@ -74,20 +75,22 @@ Teardown
     Logout MicroShift Host
 
 Read Expected Versions    # robocop: disable=too-many-calls-in-keyword
-    [Documentation]    Read ../Makefile.version.aarch64.var to find the expected versions
+    [Documentation]    Ask dnf for the version of the MicroShift package to
+    ...    find the expected versions
+    ...
     ...    Sets suite variables FULL_VERSION, MAJOR_VERSION, MINOR_VERSION, and Y_STREAM based on
     ...    the content.
 
-    # The file contains content in this format:
-    #
-    #    OCP_VERSION := 4.14.0-0.nightly-arm64-2023-05-04-012046
+    ${rc}=    SSHLibrary.Execute Command
+    ...    test -f /usr/bin/dnf
+    ...    return_rc=True
+    ...    return_stdout=False
+    ...    return_stderr=False
```

Comment:
A nit: looks like we no longer need this code.
LGTM otherwise.

Reply:
thanks, fixed